tasks/84_added-characters/added_characters.js

```diff
@@ -0,0 +1,13 @@
+export const addedChar = (str1, str2) => {
+  let str1Arr = str1.split('').sort();
+  let str2Arr = str2.split('').sort();
+  let addedLetter='';
+  for (let i = 0; i < str1Arr.length; i++) {
+    if (str1Arr[i] !== str2Arr[i]) {
+      addedLetter = str2Arr[i];
+      str2Arr.splice(i, 1);
+      i--;
```

Comment:
it is not very good to change `i` in loop and modify length of one of depended from loop arrays, and maybe it would be better to go throw bigger array as in this case there was no reason for `str2Arr[str2Arr.length - 1]`